### PR TITLE
Account settings/project management

### PR DESCRIPTION
**Pull request by vanshBhatia-A4k9**
**What this PR does / why we need it**:

- Concept for `Projects` Added
- User Guides for  `Account settings` and `Managing projects`
- Re-structured sidebar for the concept flow: `User management` -> `Projects` -> `Teaming`
- Added flow-chart for privilege level breakdown:

flow-chart-2

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Checklist:**
-   [ ] Fixes #<issue number>
-   [x] Signed the commit for DCO to be passed
-   [x] Labelled this PR & related issue with `documentation` tag

### CODE REVIEW

**Review by S-ayanide — CHANGES_REQUESTED** [review]
The content LGTM, but for the documentation writing structure can you please follow this writing style guide!

For `Concepts`
1. Intro (What is it/Definitions)
2. Pre-requisites (Links to any further knowledge required to understand this. Better to present it in a paragraph than a bulleted list)
3. Main Content - Divided into sections
4. Resources (Links to tutorials)
5. Summary (Gist of the information)
6. Learn More (Links to further sections 3-4 in order of relatedness)

For `User Guide`
1. Intro (Explain the task)
2. Before you begin (Links to Concepts covered and environment setup needed to perform this linking to get started section. Cover additional prerequisite steps if needed)
3. Need for the Task (Related Info)
4. Steps (Link to concepts wherever appropriate)
5. Resources (Related tutorials/codelabs/videos/blogs)
6. API References (Links to particular relevant sections within the API docs)
7. Learn More (Links to further sections 3-4 in order of relatedness)

This would help all the docs have a common style/appearance. If the individual section doesn't have a proper `Introduction` you can mention what the area you are writing on is about and what they can expect in this doc section.

File: website/docs/user-guides/leave-project.md [hunk 1]
```diff
@@ -4,4 +4,23 @@ title: Leave a Project
 sidebar_label: Leave Project
 ---
 
-## Coming soon
+## Introduction
+
+You can leave a project that you no longer wish to be a part of.
+
```
Comment (S-ayanide) [inline website/docs/user-guides/leave-project.md:10]: Can you add `Before you begin` for this user guide too!

File: website/docs/user-guides/my-account.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+title: My Account
+sidebar_label: My Account
+---
+
+## Introduction
+
+Under the `My Account` tab, you can change your personal details such as the email, full name, and password.
+
+> Note: The username can’t be changed as it is unique.
+
```
Comment (S-ayanide) [inline website/docs/user-guides/my-account.md:12]: A `Before you begin` section for this one as well

**Review by S-ayanide — CHANGES_REQUESTED** [review]

File: website/docs/user-guides/leave-project.md (hunk 1 above)
Comment (vanshBhatia-A4k9) [reply website/docs/user-guides/leave-project.md:10]: `change-project-name` and `leave-project` are under project management in the sidebar which is the same logical section.. this is why added it in the first section of `change-project-name` only.

File: website/docs/user-guides/my-account.md (hunk 2 above)
Comment (vanshBhatia-A4k9) [reply website/docs/user-guides/my-account.md:12]: `My account` has no relevant concept... :thinking:

**Review by S-ayanide — APPROVED** [review]

**Review by SarthakJain26 — APPROVED** [review]

**Review by arkajyotiMukherjee — APPROVED** [review]

**Review by Saranya-jena — APPROVED** [review]

**Pull request merged**